.github/workflows/main.yml

```diff
@@ -203,7 +203,7 @@ jobs:
       matrix:
         unit_type:
           - btcd unit-cover
-          - unit tags=kvdb_etcd
+          - unit tags="kvdb_etcd kvdb_postgres"
```

Comment:
Some more context: when we add the tag `kvdb_etcd` we also run channeldb unit tests on etcd.
The trick is that there's a `GetTestBackend()` function in `kvdb/backend` and there's a constant behind the tag in `kvdb/kvdb_etcd.go` and `kvdb/kvdb_no_etcd.go` that defines which backend to run. I now thing having the tags passed together works but it'll still run the channeldb tests on etcd. You may want to fix this such that those tests also run on Postgres.

Reply:
I ran the channeldb tests on postgres and they all pass except for three specialty tests. Two fail because of assumptions about the db path (does not apply to postgres) and one because of panic bubbling that is different in Postgres. Needs a closer look.

My proposal is to do that work in a separate PR where we can also evaluate the additional run time for a test suite that is already long-running. For now, I at least have manual confirmation that the channeldb tests do pass on Postgres. Let me know your thoughts.

Reply:
I've pushed the code that I used here: https://github.com/lightningnetwork/lnd/pull/5550. I think this can double as an issue?

Reply:
Isn't it the case that tacking on this extra tag doesn't actually do anything in practice? I'm guessing Go has some ordering tie-breaker here that causes it to only use the first tag? 

In any case I think we want these to run separately so we can catch distinct regressions for both backends.

Reply:
> I ran the channeldb tests on postgres and they all pass except for three specialty tests.

Are these tests still failing?

Reply:
>In any case I think we want these to run separately so we can catch distinct regressions for both backends.

They are running separately. There is `kvdb/etcd_test.go` and `kvdb/postgres_test.go`. Both build tags are set to have all code available in the test binary.